integration_tests/pages/overview/OverviewPage.ts

```diff
@@ -100,7 +100,7 @@ export default class OverviewPage extends Page {
 
   addGoalButton = (): PageElement => cy.get('#add-goal-button')
 
-  functionalSkillsTable = (): PageElement => cy.get('#functional-skills-table')
+  functionalSkillsTable = (): PageElement => cy.get('#latest-functional-skills-table')
```

Comment:
I've renamed the table from `functional-skills-table` to `latest-functional-skills-table` because we now have another table of functional skills (`all-functional-skills-table`)